p2p/config.go

```diff
@@ -52,6 +52,8 @@ type Config struct {
 	NoGossip bool
 	// Disables MDNS discovery
 	NoMdns bool
+	// Disables status exchange
+	NoStatus bool
```

Comment:
I don't think this should be exported. It should be set to true inside tests but otherwise always false.

Reply:
I am not sure how to otherwise conveniently disable status messages from the tests but I will take another look.

Reply:
fixed